bokehjs/src/coffee/main.coffee

```diff
@@ -16,7 +16,10 @@ if not window.Float64Array
 
 # binding the libs that bokeh uses so others can reference them
 Bokeh._                 = require("underscore")
-Bokeh.$ = $ = jQuery    = require("jquery")
+if $ || jQuery
+    Bokeh.$ = $ = jQuery = require("jquery").noConflict(true)
+else
+    Bokeh.$ = $ = jQuery = require("jquery")
```

Comment:
Why both? We don't want bokehjs to modify any global scope besides `window.Bokeh`.